src/libs/PaymentUtils.ts

```diff
@@ -0,0 +1,88 @@
+import {SvgProps} from 'react-native-svg';
+import {FC} from 'react';
+// eslint-disable-next-line import/no-named-default
+import {default as BankAccountModel} from './models/BankAccount';
+import getBankIcon from '../components/Icon/BankIcons';
+import CONST from '../CONST';
+import * as Localize from './Localize';
+import Fund from '../types/onyx/Fund';
+import BankAccount from '../types/onyx/BankAccount';
```

Comment:
Change these types so that `BankAccount['accountType']` is of `CONST.PAYMENT_METHODS.BANK_ACCOUNT` and 
`Fund['accountType']` is of `CONST.PAYMENT_METHODS.DEBIT_CARD`